pkg/kubelet/rkt/rkt.go

```diff
@@ -425,6 +436,68 @@ func (r *Runtime) makePodManifest(pod *api.Pod, pullSecrets []api.Secret) (*appc
 	var globalPortMappings []kubecontainer.PortMapping
 	manifest := appcschema.BlankPodManifest()
 
+	listReq := &rktapi.ListPodsRequest{
+		Filter: &rktapi.PodFilter{
+			States: []rktapi.PodState{
+				//TODO: In the future some pods can remain running after some apps exit: https://github.com/appc/spec/pull/500
+				rktapi.PodState_POD_STATE_RUNNING,
+				rktapi.PodState_POD_STATE_EXITED,
+				rktapi.PodState_POD_STATE_DELETING,
+				rktapi.PodState_POD_STATE_GARBAGE,
+			},
+			Annotations: []*rktapi.KeyValue{
+				&rktapi.KeyValue{
+					Key:   k8sRktKubeletAnno,
+					Value: k8sRktKubeletAnnoValue,
+				},
+				&rktapi.KeyValue{
+					Key:   k8sRktUIDAnno,
+					Value: string(pod.UID),
+				},
+			},
+		},
+	}
+	listResp, err := r.apisvc.ListPods(context.Background(), listReq)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't list pods: %v", err)
+	}
+
+	restartCount := 0
+	for _, rktpod := range listResp.Pods {
+		//TODO: get the manifest from listresp.Pods when this gets merged: https://github.com/coreos/rkt/pull/1786
+		inspectResp, err := r.apisvc.InspectPod(context.Background(), &rktapi.InspectPodRequest{rktpod.Id})
+		if err != nil {
+			return nil, err
+		}
+
+		if inspectResp.Pod == nil {
```

Comment:
I think it's a situation worthy of at least logging an issue, since this is a pod that at least a moment ago was a valid pod that had the kubernetes annotations, and we _should_ be able to inspect it.